hack/scale_out_poc/two_TP/nginx.conf

```diff
@@ -0,0 +1,144 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+http {
+
+    ##
+    # Basic Settings
+    ##
+
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048; 
+    proxy_http_version 1.1; 
+
+    # server_tokens off;
+
+    # server_names_hash_bucket_size 64;
+    # server_name_in_redirect off;
+
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    ##
+    # SSL Settings
+    ##
+
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+    ssl_prefer_server_ciphers on;
+
+    ##
+    # Logging Settings
+    ##
+
+    access_log /var/log/nginx/access.log;
+    error_log /var/log/nginx/error.log;
+
+        ##
+        # arktos cluster settings
+        #
+
+        server {
+                listen      {{ proxy_port }};
+                server_name {{ proxy_host_name }} {{ proxy_ip }};
+
+                access_log /var/log/nginx/arktos_access.log;
+                error_log /var/log/nginx/arktos_error.log;
+
+                set $resource_api {{ arktos_api_protocol }}://{{ resource_partition_ip }}:{{ arktos_api_port }};
+                set $tenant_api_one {{ arktos_api_protocol }}://{{ tenant_partition_one_ip }}:{{ arktos_api_port }};
+                set $tenant_api_two {{ arktos_api_protocol }}://{{ tenant_partition_two_ip }}:{{ arktos_api_port }};
+
+                #################################################################
+                # forward nodes related requests to resource partition
+                #################################################################
+
+                location ~ ^/api/v1/nodes?(.*) {
+                    proxy_pass $resource_api;
+                }
+
+                location ~ ^/apis/coordination.k8s.io/v1/leases?(.*) {
+                    proxy_pass $resource_api;
+                }
+
+                location ~ ^/apis/([^/])*/([^/])*/tenants/system/namespaces/kube-node-lease/leases?(.*) {
+                    proxy_pass $resource_api;
+                }
+
+                #################################################################
```

Comment:
Why do we need to keep the non-resource section and the non-tenant section, are they already covered by the last rule?

Reply:
You made a point. Updated as suggested.